modules/storage-persistent-storage-pv.adoc

```diff
@@ -154,14 +154,16 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 |NFS  | ✅ | ✅ | ✅
 |OpenStack Manila  | - | - | ✅
 |{rh-storage-first}  | ✅ | - | ✅
-|VMware vSphere | ✅ | - |  -
+|VMware vSphere | ✅ | - | ✅ 
```

Comment:
I think we need a ^[3]^ here as same for "ReadWriteOnce" and "AWS EBS".
Others look good to me.

Reply:
If you are talking about needing a footnote for RWX, that has been added on lines 165-166. Or are you requesting a change to that text?

Reply:
If you are talking about needing a footnote for RWX, that has been added on lines 165-166. Or are you requesting a change to that text?

Reply:
@lpettyjo I mean need a footnote in Line 157 as a pair, at least I don't see it in the preview: http://file.rdu.redhat.com/lpettyjo/OCPBUGS-1421/storage/understanding-persistent-storage.html#pv-access-modes_understanding-persistent-storage

Reply:
@duanwei33 The footnote text is there (line 165), and I've added the footnote indicator on line 157. However, for some reason, at least at the moment I'm looking at it. the preview link has not updated.

Reply:
I could see the update now, thanks.